src/api/build.ts

```diff
@@ -30,21 +31,24 @@ export async function build({
 	output = 'src/node_modules/@sapper',
 	static: static_files = 'static',
 	dest = '__sapper__/build',
+	export_build = false,
```

Comment:
I'm not particularly a fan of having this `export_build` flag to handle one type of build differently. Could we pass in the `root` instead?

Reply:
I removed it and created an extra variable pointing to the root which gets passed to the build function.